cli/command/service/rollback.go

```diff
@@ -0,0 +1,27 @@
+package service
+
+import (
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/spf13/cobra"
+)
+
+func newRollbackCommand(dockerCli *command.DockerCli) *cobra.Command {
+	options := newServiceOptions()
+
+	cmd := &cobra.Command{
+		Use:     "rollback [OPTIONS]",
+		Aliases: []string{"rollback"},
+		Short:   "Rollback a service",
+		Args:    cli.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
The pattern we use in these packages is to keep the inline function separate from adding flag.

The flags can be added at the end of `newRollbackCommand()` before `return cmd`